### PR TITLE
Repurposes desc_cult for desc_lore.

### PR DESCRIPTION
**Pull request by alsoandanswer**
Title. Frees up desc_fluff for snarky comments or whatever. Also macro-izes some spans.

Keep in mind I haven't touched desc_fluff or desc_lore in any capacity...yet.

That's for a future PR.

### CODE REVIEW

File: code/game/atoms.dm [hunk 1]
```diff
@@ -30,11 +30,11 @@
 
 	var/gfi_layer_rotation = GFI_ROTATION_DEFAULT
 
-	// Extra descriptions.
-	var/desc_fluff = null // Text about the atom's fluff description, if any exists.
+	// Extra descriptions. // Regular descriptions typically reserved for dry, raw descriptions.
+	var/desc_fluff = null // Text about the atom's fluff description, if any exists. Reserved for funny jokes and snarky comments, mostly.
```
Comment (Alberyk) [inline code/game/atoms.dm:34]: This is not how it has been used in a lot of items, that use serious lore description.

File: code/game/atoms.dm [hunk 2]
```diff
@@ -30,11 +30,11 @@
...
 	var/desc_info = null // Blue text (SPAN_NOTICE()), informing the user about how to use the item or about game controls.
 	var/desc_antag = null // Red text (SPAN_ALERT()), informing the user about how they can use an object to antagonize.
-	var/desc_cult = null // Purple text (SPAN_CULT()), telling the user, if they're a cultist, how they can use certain items related to being a cultist.
+	var/desc_lore = null // Purple text (SPAN_CULT()) that gives the atom's lore info, be it species, corporate or whatever.
```
Comment (Alberyk) [inline code/game/atoms.dm:37]: This is not used anywhere and it is the same as desc_fluff, should be removed.

**Review by Alberyk — CHANGES_REQUESTED** [review]

File: code/game/atoms.dm (hunk 1 above)
Comment (alsoandanswer) [reply code/game/atoms.dm:34]: The problem I'm trying to fix is that desc_fluff is both used for serious lore description and jokes.
With this PR I'm planning to seperate them out, so serious lore uses desc_lore and jokes use desc_fluff.

Comment (SleepyGemmy) [inline code/game/atoms.dm:34]: this is not true and is a subjective appropiation of the variable.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
jokes don't belong in the descriptions of items, since they're descriptions.
if something is more of a quip or humorous observation, then it could fit in desc_fluff (see the example below).
the reason a cultist description exists is because only cultists can do certain interactions with certain items, not every antagonist.

for reference:
`desc` is for shorthand descriptions of items. example: "A lightbulb made out of glass."
`desc_fluff` is for longer descriptions of items, extra information, and/or lore. example: "This particular lightbulb is manufactured by the Idris Incorporation for use in service installations across the galaxy. Commonly thought to be of cheap quality."
`desc_antag` is for antagonism descriptions.
`desc_cult` is for cultist related descriptions.

**Comment by SleepyGemmy** [comment]
i'd rather see it left as-is and a `desc_humour`/`desc_joke` variable added instead.

**Comment by alsoandanswer** [comment]
!review

**Comment by SleepyGemmy** [comment]
closing as PR 14843 was merged.

**Pull request closed without merging**